Firestore/core/test/firebase/firestore/util/async_queue_test.cc

```diff
@@ -86,15 +86,18 @@ TEST_P(AsyncQueueTest, VerifyIsCurrentQueueWorksWithOperationInProgress) {
 TEST_P(AsyncQueueTest, CanScheduleOperationsInTheFuture) {
   std::string steps;
 
+  // TODO(varconst): these tests used to check that operations are executed
+  // according to the time for which they're scheduled, not in the order they
+  // were added. Restore this check, probably by using a fake clock.
```

Comment:
I'm really surprised that the behavior has changed so radically that the old test no longer works.

When the test fails what behavior are you seeing? The scheduled actions running sooner than the immediate ones?